src/commands/test/util/browser-tests-runner/index.js

```diff
@@ -173,21 +180,23 @@ exports.run = function(allTests, options, devTools) {
     return startServer(filteredTests, options, devTools)
         .then((result) => {
             console.log(`Running "${result.url}" using mocha-phantomjs...`);
-            var mochaPhantomJSOptions = { useColors:true };
+            var mochaPhantomJSOptions = result.phantomOptions || { useColors: true };
 
-            if(shouldCover) {
+            if (shouldCover) {
                 mochaPhantomJSOptions.hooks = 'mocha-phantomjs-istanbul';
                 mochaPhantomJSOptions.coverageFile = getCoverageFile();
             }
 
             return spawn(phantomjsBinPath, [mochaPhantomJSBin, result.url, 'spec', JSON.stringify(mochaPhantomJSOptions)], {
                 stdio: 'inherit'
             }).then(function() {
-                result.stopServer();
+                process.exit(0);
```

Comment:
I see another `process.exit(1)` in `server-tests-runner` for the catch block, so maybe that's normal for these testing frameworks? But is exiting when it succeeds normal as well?